pkg/api/v1/atlasdeployment_types.go

```diff
@@ -304,10 +307,28 @@ type ProcessArgs struct {
 
 func (specArgs ProcessArgs) ToAtlas() (*mongodbatlas.ProcessArgs, error) {
```

Comment:
Why do we use value receiver here? Autogenerated methods use pointer receiver. I might be a bit confusing